app/src/androidTest/java/net/kuama/documentscanner/extensions/MatOfPointExtensionInstrumentedTest.kt

```diff
@@ -0,0 +1,34 @@
+package net.kuama.documentscanner.extensions
+
+import org.junit.Assert.assertEquals
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import net.kuama.documentscanner.utils.TestUtils
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.opencv.core.MatOfPoint
+import org.opencv.core.Point
+
+@RunWith(AndroidJUnit4::class)
+class MatOfPointExtensionInstrumentedTest {
```

Comment:
It seems more a unit test, does it run on an emulator?
```suggestion
class MatOfPointExtensionTest {
```

Reply:
Yes. It has to run on the emulator otherwise this test cannot be successful. To load OpenCv we need the context, which is present only on instrumented test.